docs/network/wwivnet_linux.md

```diff
@@ -1,72 +1,20 @@
 # Configuring WWIVnet on Linux
+
 ***
 
-Setting up WWIVnet and subscribing to Subs isn't hard, but there are a few 
-gotchas; mostly due to filename inconsistencies and having to use DOSEMU 
-for some of the tools (NET38, et al)
-
-**STEP #1**  - Get a WWIVnet Node Number assigned.  
-You will need the Node Number for several steps in this setup.  
-**STEP #2**  - send a password to 1@1 for the binkp auth.
-You can do this at the same time as getting the Node Number.  You will need this for your callout.net file later.
-
-**NOTE:** Please don't bother to try configuring things ahead of time; 
-it will just cause issues during setup.  
-
-_Where Stuff Goes_  
-This is some layout info so you have a handle on where everything belongs. 
-We will build up the associated structure as we go along. The surrounded by 
-"** **" are the ones specific to WWIVnet.
-
-    ${WWIV_DIR}
-        .dosemurc
-        .wwivrc
-        in.nodemgr
-        **network0**
-        **network1** (symbolic link to network0)
-        **network2** (symbolic link to network0)
-        **network3** (symbolic link to network0)
-        data
-        nets
-            **wwivnet**    
-        .dosemu
-            drive_c
-                autoexec.bat
-                **network1.bat**
-                **network2.bat**
-                **network3.bat**
-
-### Installing Binaries
-
-1. Install NET38b6.ZIP binaries
-* unzip NET38b6.ZIP in ${WWIV_DIR}
-
-### Configuring WWIVnet
-
-**WWIVnet under dosemu**
-
-Since we need to run the NET38 binaries under dosemu, we will 
-get the dosemu parts going first. For a detailed explanation of how 
-to set up DOSEMU (in particular, the relationship between shell 
-scripts and batch files), check out [DOSEMU common settings] first. 
-We are going to talk about only the specifics of DOSEMU in the context 
-of WWIVnet in this section. We assume you already have DOSEMU running 
-successfully (have your .dosemurc configured and have manually run 
-dosemu).
-
-_dosemu config_
-
-1. copy the *.bat files from ${WWIV_DIR}/dosemu-batchfiles to your dosemu C: drive (e.g, ~/.dosemu/drive_c)
-2. edit files to suit your system (install.sh should have taken care of this already
-but if not, look for any occurance of "REPLACE-WWIVBASE" in the network*.bat files
-* make sure paths match your setup (e.g, in batch files)
-
-_network scripts_
+These directions supercede and replace the general directions for linux from 
+"Configuring WWIVnet" onward.  You absolutely have to get a WWIVnet node number 
+assigned; 1@1 will supply the password and node number.
+
+It's best to get that information **before** attempting to set up WWIVnet.
+
+Throughout, ${WWIV_DIR} refers to your WWIV base installation directory.
+    
+## Installing Network Scripts
 
 We have a common network shell script that can be used to reference 
```

Comment:
Hmm, I don't use this on linux actually. I wonder if all of this shell script is not needed anymore by anyone?

Reply:
Oh!  Um, I mean, I followed the directions and it worked, so I presumed it was needed.  LOL.   If it's not needed, then yeah, let's take it out.